src/bundle/Resources/views/themes/admin/ui/form_fields/dropdown_widget.html.twig

```diff
@@ -17,7 +17,8 @@
             multiple: form.vars.multiple,
             custom_init: attr.dropdown_custom_init|default(false),
             class: attr.dropdown_class|default(''),
-            translation_domain: choice_translation_domain|default(false),
+            choice_translation_domain: choice_translation_domain|default(false),
```

Comment:
Don't we have a default already within `choice_translation_domain` variable?

Reply:
It follows the established pattern. Additionally in theory Twig configured with `strict_variables` option (which should be used in dev env, but sometimes it's just `null`) might complain that we're using a variable that does not exist.